lib/cgrates_web_jsonapi/tariff_plans/tp_route.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CgratesWebJsonapi.TariffPlans.TpRoute do
+  use Ecto.Schema
+  import Ecto.Changeset
+  use EctoConditionals, repo: CgratesWebJsonapi.Repo
+
+  @attributes ~w[tpid route_id route_weight tp_route_id route_weight tenant sorting sorting_parameters
+   filter_ids activation_interval route_filter_ids route_account_ids route_resource_ids
+   route_ratingplan_ids route_stat_ids route_blocker route_parameters weight]a
+
+  @primary_key {:pk, :id, autogenerate: true}
+  schema "tp_routes" do
+    field :id, :integer, source: :pk
+    field :route_id, :string, source: :id
```

Comment:
let's rename this field in order to avoid confusions with `route_id` field
```elixir
field :custom_id, :string, default: ""
field :route_id, default: ""
```